src/main/java/micdoodle8/mods/galacticraft/core/nei/NEIGalacticraftConfig.java

```diff
@@ -140,121 +134,56 @@ public void registerRecipes() {
+         * PositionedStack( GT_OreDictUnificator.get(OrePrefixes.screw, Materials.StainlessSteel, 1), 26 - x, 37 - y));
+         * input.put( 19, new PositionedStack( GT_OreDictUnificator.get(OrePrefixes.screw, Materials.StainlessSteel, 1),
+         * 98 - x, 37 - y)); input.put( 20, new PositionedStack( GT_OreDictUnificator.get(OrePrefixes.screw,
+         * Materials.StainlessSteel, 1), 116 - x, 37 - y)); input.put( 21, new PositionedStack(
+         * GT_OreDictUnificator.get(OrePrefixes.screw, Materials.StainlessSteel, 1), 8 - x, 91 - y)); input.put( 22, new
+         * PositionedStack( GT_OreDictUnificator.get(OrePrefixes.screw, Materials.StainlessSteel, 1), 26 - x, 91 - y));
+         * input.put( 23, new PositionedStack( GT_OreDictUnificator.get(OrePrefixes.screw, Materials.StainlessSteel, 1),
+         * 98 - x, 91 - y)); input.put( 24, new PositionedStack( GT_OreDictUnificator.get(OrePrefixes.screw,
+         * Materials.StainlessSteel, 1), 116 - x, 91 - y)); input.put(25, new PositionedStack(new
+         * ItemStack(GCItems.heavyPlatingTier1), 44 - x, 37 - y)); input.put(26, new PositionedStack(new
+         * ItemStack(GCItems.heavyPlatingTier1), 62 - x, 37 - y)); input.put(27, new PositionedStack(new
+         * ItemStack(GCItems.heavyPlatingTier1), 80 - x, 37 - y)); input.put(28, new PositionedStack(new
+         * ItemStack(GCItems.heavyPlatingTier1), 44 - x, 55 - y)); input.put(29, new PositionedStack(new
+         * ItemStack(GCItems.heavyPlatingTier1), 80 - x, 55 - y)); input.put(30, new PositionedStack(new
+         * ItemStack(GCItems.heavyPlatingTier1), 44 - x, 73 - y)); input.put(31, new PositionedStack(new
+         * ItemStack(GCItems.heavyPlatingTier1), 80 - x, 73 - y)); input.put(32, new PositionedStack(new
+         * ItemStack(GCItems.heavyPlatingTier1), 44 - x, 91 - y)); input.put(33, new PositionedStack(new
+         * ItemStack(GCItems.heavyPlatingTier1), 62 - x, 91 - y)); input.put(34, new PositionedStack(new
+         * ItemStack(GCItems.heavyPlatingTier1), 80 - x, 91 - y)); this.registerBuggyBenchRecipe(input, new
+         * PositionedStack(new ItemStack(GCItems.buggy), 143 - x, 64 - y)); HashMap<Integer, PositionedStack> input2 =
+         * new HashMap<>(input); input2.put(35, new PositionedStack(RecipeUtil.getChestItemStack(1, 3), 107 - x, 64 -
+         * y)); this.registerBuggyBenchRecipe(input2, new PositionedStack(new ItemStack(GCItems.buggy, 1, 1), 143 - x,
+         * 64 - y)); input2 = new HashMap<>(input); input2.put(35, new PositionedStack(RecipeUtil.getChestItemStack(1,
+         * 0), 107 - x, 64 - y)); this.registerBuggyBenchRecipe(input2, new PositionedStack(new ItemStack(GCItems.buggy,
+         * 1, 2), 143 - x, 64 - y)); input2 = new HashMap<>(input); input2.put(35, new
+         * PositionedStack(RecipeUtil.getChestItemStack(1, 1), 107 - x, 64 - y)); this.registerBuggyBenchRecipe(input2,
+         * new PositionedStack(new ItemStack(GCItems.buggy, 1, 3), 143 - x, 64 - y));
+         */
```

Comment:
remove the dead code, git history is here if it is needed again.

Reply:
Shall we remove other dead code?

Reply:
yes, unless there is explicitely a comment saying why it sho